dde-store_tr.ts

```diff
@@ -0,0 +1,269 @@
+            <source>Internet</source>
+            <translation>İnternet</translation>
+        </message>
+        <message>
+            <location filename="../mainwindow.cpp" line="118"/>
+            <source>Games</source>
+            <translation>Oyunlar</translation>
+        </message>
+        <message>
+            <location filename="../mainwindow.cpp" line="119"/>
+            <source>Development</source>
+            <translation>Geliştirme</translation>
+        </message>
+        <message>
+            <location filename="../mainwindow.cpp" line="120"/>
+            <source>Office</source>
+            <translation>Ofis</translation>
+        </message>
+        <message>
+            <location filename="../mainwindow.cpp" line="121"/>
+            <source>Graphics</source>
+            <translation>Grafikler</translation>
+        </message>
+        <message>
+            <location filename="../mainwindow.cpp" line="122"/>
+            <source>Video</source>
+            <translation>Video</translation>
+        </message>
+        <message>
+            <location filename="../mainwindow.cpp" line="123"/>
+            <source>Music</source>
+            <translation>Müzik</translation>
+        </message>
+        <message>
+            <location filename="../mainwindow.cpp" line="124"/>
+            <source>System</source>
+            <translation>Sistem</translation>
+        </message>
+        <message>
+            <location filename="../mainwindow.cpp" line="125"/>
+            <source>Installed</source>
+            <translation>Yüklenenler</translation>
+        </message>
+        <message>
+            <location filename="../mainwindow.cpp" line="230"/>
+            <source>Cannot close while app is being installed</source>
+            <translation>Uygulama yüklenirken kapatılamaz</translation>
+        </message>
+        <message>
+            <location filename="../mainwindow.cpp" line="231"/>
+            <source>OK</source>
+            <translation>Tamam</translation>
+        </message>
+    </context>
+    <context>
+        <name>CategoryPage</name>
+        <message>
+            <location filename="../pages/categorypage.cpp" line="15"/>
+            <source>Sort by</source>
+            <translation>Sıralama türü</translation>
+        </message>
+        <message>
+            <location filename="../pages/categorypage.cpp" line="18"/>
+            <source>Alphabetical</source>
+            <translation>Alfabetık</translation>
+        </message>
+        <message>
+            <location filename="../pages/categorypage.cpp" line="19"/>
+            <source>Ratings</source>
+            <translation>Derecelendirmeye göre</translation>
+        </message>
+        <message>
+            <location filename="../pages/categorypage.cpp" line="39"/>
+            <source>No results for %1</source>
+            <translation>%1 için sonuç bulunamadı/translation>
```

Comment:
Missing the < in the closing </translation> tag.

Reply:
Corrected. Thank you.